tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Telemetry.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="ConfigurationKeys.Telemetry.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Telemetry;
+
+namespace Datadog.Trace.Configuration
+{
+    internal static partial class ConfigurationKeys
+    {
+        internal static class Telemetry
+        {
+            /// <summary>
+            /// Configuration key for enabling or disabling internal telemetry.
+            /// Default value is <c>true</c> (enabled).
+            /// </summary>
+            public const string Enabled = "DD_INSTRUMENTATION_TELEMETRY_ENABLED";
+
+            /// <summary>
+            /// Configuration key for sending telemetry direct to telemetry intake. If enabled, and
+            /// <see cref="ConfigurationKeys.ApiKey"/> is set, sends telemetry direct to intake if agent is not
+            /// available. Enabled by default if <see cref="ConfigurationKeys.ApiKey"/> is available.
+            /// </summary>
+            public const string AgentlessEnabled = "DD_INSTRUMENTATION_TELEMETRY_AGENTLESS_ENABLED";
```

Comment:
Do you plan to do the renaming discussed on Slack in a different PR?

Reply:
yep!